### ISSUE TITLE
inconsistent requrement

### ISSUE THREAD

**Issue by moinfar** [theislab/dca#13]
Hi,

Recent updates on scanpy made this package inconsistent with it.
If you use scanpy>=1.1a (given an arbitrary input), on saving stage you will face:
```
  File "/usr/local/lib/python3.5/site-packages/DCA/train.py", line 168, in train_with_args
    net.predict(adata, predict_columns)
  File "/usr/local/lib/python3.5/site-packages/DCA/network.py", line 404, in predict
    res = super().predict(adata, colnames=colnames, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/DCA/network.py", line 220, in predict
    sc.write('output', adata)
  File "/usr/local/lib/python3.5/site-packages/scanpy/readwrite.py", line 159, in write
    if is_valid_filename(filename):
  File "/usr/local/lib/python3.5/site-packages/scanpy/readwrite.py", line 510, in is_valid_filename
    elif ext[-1][1:] in avail_exts:
IndexError: list index out of range
```

The source of problem is [this commit](https://github.com/theislab/scanpy/commit/6e789a6bb327339c4c7caeee98b401336675ff28) of scanpy.

Regards :D

**Comment by gokceneraslan** [comment]
Thanks a lot for the bug report. It's actually [the next commit](https://github.com/theislab/scanpy/commit/cd41d16b2636dc61d63fcc113fff96e1089b5a5e) that broke this. I informed Alex there, thanks again!

**Comment by gokceneraslan** [comment]
Fixed in 0.2.
